growth/too/views.py

```diff
@@ -238,11 +241,114 @@ def event(dateobs):
         'event.html', event=models.Event.query.get_or_404(dateobs))
 
 
-@app.route('/event/<datetime:dateobs>/objects')
+@app.route('/event/<datetime:dateobs>/objects', methods=['GET', 'POST'])
 @login_required
 def objects(dateobs):
+    """Objects page"""
+
+    subquery_time = models.db.session().query(
+        models.Lightcurve.name
+        ).group_by(models.Lightcurve.name).\
```

Comment:
Once you have fixed the hybrid_property issue, you should be able to replace this query with simply:

    candidates = models.Candidates.query.filter(models.Candidates.first_detection_time > datetime)